pallets/validators-manager/src/tests/mock.rs

```diff
@@ -405,7 +405,7 @@ parameter_types! {
     pub const ErasPerGrowthPeriod: u32 = 2;
     pub const RewardPaymentDelay: u32 = 2;
     pub const RewardPotId: PalletId = PalletId(*b"av/vamgr");
-    pub const MaxCandidates: u32 = 257;
+    pub const MaxCandidates: u32 = 253;
```

Comment:
this magic number is changed. Now it is less than the limit on Validators Manager